pom.xml

```diff
@@ -75,7 +75,7 @@
         <dependency>
           <groupId>com.fasterxml.jackson.core</groupId>
           <artifactId>jackson-databind</artifactId>
-          <version>2.9.10.3</version>
+          <version>2.8.11</version>
```

Comment:
https://github.com/awslabs/amazon-kinesis-agent/blob/master/setup#L110

reverted because of this